htdocs/banners.php

```
@@ -275,55 +298,55 @@ function emailStats($cid, $bid)
 {
     /** @var \XoopsMySQLDatabase $xoopsDB */
     global $xoopsDB, $xoopsConfig;
-    $banner_login = isset($_SESSION['banner_login']) ? (string) $_SESSION['banner_login'] : '';
-    $banner_pass  = isset($_SESSION['banner_pass']) ? (string) $_SESSION['banner_pass'] : '';
-    if ($banner_login != '' && $banner_pass != '') {
+    $banner_client_id = isset($_SESSION['banner_client_id']) ? (int) $_SESSION['banner_client_id'] : 0;
```

PHPMD (2.15.0)

[error] The variable $banner_client_id is not named in camelCase. (undefined)

(CamelCaseVariableName)